api/src/controllers/publications.js

```diff
@@ -225,7 +246,7 @@ async function scrapeGoogleScholar(url) {
   const publicationInfo = {};
   fields.forEach((key, i) => (publicationInfo[key] = values[i]));
 
-  // TODO: this logic depends on the order of the fields, 
+  // TODO: this logic depends on the order of the fields,
   // which will differ based on the info of the publication, can be improved
```

Comment:
Can you be more descriptive with what specifically needs to be done in this `TODO`? In case someone else wants to pick it up

Reply:
It's a bit complicated so I'm going to explain it in a ticket instead, and it's probably not an issue until someone reports it as a bug with specific publications.

Reply:
SG